specs/select.spec.md

```diff
@@ -0,0 +1,156 @@
+# Select + Select-Option
+
+## Overview
+
+The `select` is a component that provides a list of options for the user to select from.
+
+### Background
+
+There is a standard select html control, `<select>` and `<option>`, but it doesn't support the full level of detail, styling, etc that most design systems need or require. For context please see [MDN Docs](https://developer.mozilla.org/en-US/docs/web/html/element/select).
+
+### Use Cases
+
+- *A customer using the component on a web page.*
+On a web page a customer is shopping for a new shirt in size medium. The customer opens the size `select` and sees a list of `<option>`s and selects size medium.
+
+### Features
+- form association
+- focus delegation
+
+### Features
+- Select one or multiple `<fast-option/>`(s) using the `multiple` attribute
+- When Dropdown does not have enough screen real estate to open below `<fast-select>` it will open above.
```

Comment:
Do we want more positioning choices (ie. always open above)? Leaving that to later version works too.

Reply:
Have you thought about how the flyout menu is placed in the dom, breaks out of parent containers or not, positioning mode?

Reply:
I'm open here, but thinking it would just use the intersection observer work you have done.

Reply:
Positioning choices (left/right/above/below/responsive/etc...) can be deferred until later.  Breaking out of parent containers is a different and more difficult problem.  Only approach I've come up with would use fixed positioning to position the menu, but the side effect of that is that the open menu no longer tracks the button if the page scrolls.  I suppose we could add that or something like it as a feature later as well.